stack-8.0.yaml

```diff
@@ -6,6 +6,7 @@ packages:
 - proto-lens-combinators
 - proto-lens-optparse
 - proto-lens-tests
+- proto-lens-benchmarks
```

Comment:
Can you also add this to the `PACKAGES` list in `travis-cabal.sh`?  (Sorry that this list is duplicated in multiple places...)